Added Overview page & edited Getting Started page

nimraahmed opened the pull request:
@FelixMalfait Added Overview page & edited Getting Started page to the documentation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/twentyhq/twenty?pullRequest=1875) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/twentyhq/twenty?pullRequest=1875) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/twentyhq/twenty?pullRequest=1875) it.</sub>

FelixMalfait commented on docs/docs/start/getting-started.mdx:
```diff
@@ -1,19 +1,249 @@
+sidebar_position: 1
 sidebar_custom_props:
-  icon: TbBolt
+  icon: TbDeviceDesktop
```
Isn't TbDeviceDesktop already used elsewhere? I didn't check but assume it would be for local setup. Make sure it isn't used twice, that would be ugly :)

FelixMalfait commented on docs/docs/start/getting-started.mdx:
```diff
@@ -1,19 +1,249 @@
+**Note:** `npm` currently does not support local packages satisfactorily. We strongly recommend using `yarn` instead.
 
-If you are looking to install the project locally, either to try it or to contribute, check out our [developer guide](/developer/local-setup).
+## Prerequisites
```
Did you copy/paste the content from another page? We'd like to avoid duplicate content from one page to another.
Maybe you planned to remove it after? We try not to merge PR in an intermediate state. If you raise a PR it's better if it doesn't break anything or introduce any bad pattern

FelixMalfait commented on docs/package-lock.json:
Please use yarn exclusively and don't introduce this file

FelixMalfait commented on docs/docs/start/overview.mdx:
```diff
@@ -0,0 +1,38 @@
+
+Twenty is a modern CRM offering the flexibility of open source, advanced features, and a sleek design that puts teams in full control. 
+
+Twenty's codebase is adaptable, fostering a collaborative environment where every user can contribute. This extensible and modular design ensures our product evolves with you, cultivating mutually beneficial growth that promotes progress and innovation.
```
I'm sorry I know we wrote all of this text in README (probably I committed it), but I really don't like it anymore haha. 
I'll send you things we can take inspiration from in Discord

FelixMalfait reviewed (commented):
Thanks!!!

nimraahmed commented:
**Changes made in recent commits:**

- Added an overview page with more details about the product
- Fixed edit link
- Moved hosting above developer guide 
- Moved glossary to additional resources 
- Revised project readme
- Cleaned up and edited local setup page in the docs to make steps clearer & more structured 
- Deleted sweep.yaml, and .ergomake and .devcontainer folders

FelixMalfait commented on README.md:
```diff
@@ -80,6 +87,15 @@ Below are the features we have implemented to date.
     </picture>
 </p>
 
+<p align="center">
```
No title? For consistency. Design might be a bit odd otherwise no?

FelixMalfait commented on README.md:
```diff
@@ -90,44 +106,32 @@ Below are the features we have implemented to date.
-    <img alt="Book us with Cal.com" src="https://cal.com/book-with-cal-light.svg" height="40">
-  </picture>
-</a>
+Welcome to the future of CRM. Welcome to Twenty.
```
Maybe I'd get rid of this as it's not necessary and a bit too much marketing speak

FelixMalfait commented on README.md:
```diff
@@ -90,44 +106,32 @@ Below are the features we have implemented to date.
-# Office hours
-Want to meet the team? Get direct support?
-Feel free to book a slot during our office hours.
+🚀 Get Started with [Twenty](https://docs.twenty.com/developer/local-setup). 
```
Link on full CTA (including get started...)?

FelixMalfait commented on docs/docs/developer/_category_.json:
```diff
@@ -1,4 +1,4 @@
 {
   "label": "Developer guide",
```
Should we rename it? Maybe you've planned it for future steps with other changes we have discussed. But we need to clarify contributing VS extending the CRM which isn't the same

FelixMalfait commented on README.md:
```diff
@@ -7,11 +7,12 @@
+<p align="center">CRM flexibility, tailored to your unique business needs</p>
 
-<p align="center">🌐 <a href="https://twenty.com">Website</a> · 📚 <a href="https://docs.twenty.com">Documentation</a> · 📖 <a href="https://twenty.com/story">Our Story</a> · 💬 <a href="https://discord.gg/cx5n4Jzs57">Discord</a> · ✏️ <a href="https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty">Figma</a><p>
+<p align="center">🌐 <a href="https://twenty.com">Website</a> · 📚 <a href="https://docs.twenty.com">Documentation</a> · 💻 <a href="https://app.twenty.com/">Demo</a> · 💬 <a href="https://discord.gg/cx5n4Jzs57">Discord</a> · ✏️ <a href="https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty">Figma</a><p>
```
can we replace the link to Discord by twenty.com/discord? nicer

FelixMalfait reviewed (commented):
Great!
